### PR TITLE
Fix for batch size > 1 for reshape/shape layers

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
@paveltc This is the PR for which you need to run the smoke test

### CODE REVIEW

File: model_compiler/python/nnir_to_nnef.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def main():
     inputFolder = sys.argv[1]
     outputFolder = sys.argv[2]
     print('reading NNIR model from ' + inputFolder + ' ...')
-    graph = IrGraph()
+    graph = IrGraph(True)
```
Comment (kiritigowda) [inline model_compiler/python/nnir_to_nnef.py:130]: Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)

**Comment by kiritigowda** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           

Complexity increasing per file
==============================
- model_compiler/python/nnir.py  2
         

Clones added
============
- model_compiler/python/nnir_to_clib.py  1
- model_compiler/python/nnir_update.py  1
- model_compiler/python/nnir_to_openvx.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497&bid=18837277)

File: model_compiler/python/onnx_to_nnir.py [hunk 2]
```diff
@@ -157,7 +157,7 @@ def onnx_value_info_to_data(info, dims):
     return tensor
 
 def onnx_graph_to_ir_graph(onnx_graph):
-    graph = IrGraph()
+    graph = IrGraph(False)
```
Comment (kiritigowda) [inline model_compiler/python/onnx_to_nnir.py:160]: Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)

File: model_compiler/python/nnir_update.py [hunk 3]
```diff
@@ -59,7 +59,7 @@ def main():
     inputFolder = sys.argv[pos]
     outputFolder = sys.argv[pos+1]
     print('reading IR model from ' + inputFolder + ' ...')
-    graph = IrGraph()
+    graph = IrGraph(True)
```
Comment (kiritigowda) [inline model_compiler/python/nnir_update.py:62]: Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)

File: model_compiler/python/nnef_to_nnir.py [hunk 4]
```diff
@@ -236,7 +236,7 @@ def nnef_op_to_ir_node(nnef_graph, nnef_operation):
     
 def nnef_graph_to_ir_graph(nnef_graph):
 
-    graph = IrGraph()
+    graph = IrGraph(False)
```
Comment (kiritigowda) [inline model_compiler/python/nnef_to_nnir.py:239]: Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)

File: model_compiler/python/nnir_to_clib.py [hunk 5]
```diff
@@ -1975,7 +1975,7 @@ def main():
     inputFolder = sys.argv[pos]
     outputFolder = sys.argv[pos+1]
     print('reading IR model from ' + inputFolder + ' ...')
-    graph = IrGraph()
+    graph = IrGraph(True)
```
Comment (kiritigowda) [inline model_compiler/python/nnir_to_clib.py:1978]: Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)

File: model_compiler/python/caffe_to_nnir.py [hunk 6]
```diff
@@ -854,7 +854,7 @@ def extractCaffeNodeInfo(net_parameter, graph, inputsInfo, verbose):
 
 # convert caffe graph to ir graph.
 def caffe_graph_to_ir_graph(net_parameter, input_dims, verbose):
-    graph = IrGraph()
+    graph = IrGraph(False)
```
Comment (kiritigowda) [inline model_compiler/python/caffe_to_nnir.py:857]: Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)

**Comment by paveltc** [comment]
@LakshmiKumar23 @kiritigowda This PR passes the smoke tests.

**Comment by LakshmiKumar23** [comment]
@kiritigowda Resolved issues after we spoke yesterday about the * imports

**Comment by LakshmiKumar23** [comment]
@rrawther Rajy, @paveltc ran the smoke test on this PR

**Pull request merged**